### PR TITLE
Remove rack dependency 

### PR DESCRIPTION
**Pull request by downgba**
In order to make the gem compatible with new rails version, it was necessary to upgrade rack gem

### CODE REVIEW

**Comment by duduribeiro** [comment]
but why we are dependning on Rack? 🤔 
cc @fpgentil

**Comment by duduribeiro** [comment]
Can we remove this rack dependency?

**Comment by duduribeiro** [comment]
And, if it is needed, I think that we can rely on old rack versions too.

Instead of making `spec.add_dependency 'rack', '~> 2.0', '>= 2.0'`

You can try:

```
spec.add_dependency 'rack', '>= 1.6.4'
```

With this, we allow `2.0` and keep ~> 1.6.4 working

**Comment by duduribeiro** [comment]
This change is breakable for non rack 2.0 users.

**Comment by downgba** [comment]
we can remove it from gemspec, it is a savon dependency.

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,6 @@ language: ruby
 before_install:
 - gem install bundler -v '~> 1.5'
 rvm:
-- 2.1.10
```
Comment (duduribeiro) [inline .travis.yml]: why did you remove this? 
I think it is a good idea to test on `2.1.10`. WDYT?

**Comment by duduribeiro** [comment]
LGTM 👍

File: .travis.yml (hunk 1 above)
Comment (downgba) [reply .travis.yml]: because there is no support for this version

Comment (downgba) [reply .travis.yml]: this version was discontinued
